pkg/command/map_string.go

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 Authors of Cilium
+
+package command
```

Comment:
I look around, and find this package is the most relevant, feel free to let me know otherwise.

Reply:
We also have `tools/`, what do you think?

Reply:
tools directory is used for extra utility (e.g. lint check, dev-doctor, etc), should not be used in any main application code.